nautobot/core/jobs/__init__.py

```diff
@@ -361,6 +371,131 @@ def run(self, *, content_type, csv_data=None, csv_file=None, roll_back_if_error=
             raise RunJobTaskFailed("CSV import not fully successful, see logs")
 
 
+def get_data_compliance_rules():
+    """Generate a list of Audit Ruleset classes that exist from the registry."""
+    validators = []
+    for rule_sets in get_data_compliance_rules_map().values():
+        validators.extend(rule_sets)
+    return validators
+
+
+def get_choices():
```

Comment:
Can we give this a less generic name? `get_data_compliance_choices` or something?